README.md

```diff
@@ -138,8 +128,20 @@ You can deploy this project using:
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
-
-👤 **Author: Damilare**
+ 
+ 👤 **Ouail Laarabi**
+ 
+ - GitHub: [Click here](https://github.com/Ouail-01?tab=overview&from=2023-01-01&to=2023-01-02)
+ - Twitter: [Click here](https://twitter.com/Ouail_Laarabi)
+ - LinkedIn: [Click here](https://www.linkedin.com/in/ouail-laarabi-53203b250/)
+ 
+ 👤 **Prince**
+ 
+ - GitHub: [@prius2055](https://github.com/prius2055)
+ - Twitter: [@prius2055](https://www.twitter.com/prius2055)
+ - LinkedIn: [@prius2055](https://www.linkedin.com/prius2055)
+

```

Comment:
![image](https://github.com/Bestbynature/Morse-Code-Interpreter/assets/93039359/104a304c-8c3e-4691-a003-01606fd724da)

- [ ] It would be nice if you fix this highlighted LinkedIn handle because it is showing me an error which you can see below ⬇️ Kindly fix this link to make your readme descriptive and professional 👍
![image](https://github.com/Bestbynature/Morse-Code-Interpreter/assets/93039359/a62df3dd-5aba-40ba-9b79-3920c5524c53)